src/main/java/reposense/git/GitBlamer.java

```diff
@@ -11,24 +8,19 @@
 
 public class GitBlamer {
 
-    private static final Pattern INSERTION_PATTERN = Pattern.compile("^author (.+)");
+    private static final int AUTHOR_NAME_OFFSET = 7;
```

Comment:
Do you think if we put "author ".length() would be better?

Because I think its more explanatory that way